Update creating-a-default-community-health-file.md (#34270)
github/docs: content/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file.md

## About default community health files

Default community health files are a set of predefined files that provide guidance and templates for maintaining a healthy and collaborative open source project. These files help you automate and standardize various aspects of your project's development and community interaction, promoting transparency, good practices, and collaboration.

> You can add default community health files to a public repository called `.github`. {% data variables.product.product_name %} will use and display default files for any repository owned by the account that does not have its own file of that type. The location of these default files determines whether and how they will be used.

Reviewer: ```suggestion
You can add default community health files to a public repository called `.github` and {% data variables.product.product_name %} will use and display default files for any repository owned by the account that does not have its own file of that type, or create them directly in the relevant repository for higher specificity.
```

> ### File Location and Precedence

Reviewer: I'd either skip the heading or move it after "… good practices, collaboration" before starting to talk about any actual files and their locations.

When a repository does not contain its own file of a specific type, {% data variables.product.product_name %} will look for a default file in the following order:

> 1. **Within the Local Repository**:
>    1. `.github` folder
>    1. Root of the repository
>    1. `docs` folder
> 1. **Within the `.github` Repository**:
>    1. `.github` folder
>    1. Root of the repository
>    1. `docs` folder

Reviewer: I'm not sure "Local" is a correct term here when git is a distributed system — I'd leave that to content triage folks to chime in with some better recommendation (see the original wording how it carefully avoids using any other phrase than just "repository" in pointing to the current repo?)

Maybe to improve scanability:

```suggestion
 1. `.github` folder in the project's repository
 1. Root of the repository
 1. `docs` folder of the project's repository
 1. `.github` folder in the `.github` repository
 1. Root of the `.github` repository
 1. `docs` folder of the `.github` repository
```

to explicitly show the lookup is completely linear, not either–or i.e. repo/docs still precedes .github/.github …

> If no corresponding file is found in the local repository, {% data variables.product.product_name %} will use the default file from the `.github` repository, following the same order of precedence.
> 
> For example, if someone creates an issue or pull request in a repository that does not have its own `CONTRIBUTING.md` file, {% data variables.product.product_name %} will check the local repository first and, if no file is found, it will fall back to the default `CONTRIBUTING.md` in the `.github` repository.
> 
> However, if a repository contains any files in its own `.github/ISSUE_TEMPLATE` folder, such as issue templates or a `_config.yml` file, none of the contents of the default `.github/ISSUE_TEMPLATE` folder in the `.github` repository will be used. This allows repository maintainers to override the default files with specific templates or configurations as needed.

Reviewer: to simplify already stated:

```suggestion
For example, anyone who creates an issue or pull request in a repository that does not have its own `CONTRIBUTING.md` file will see a link to the default `CONTRIBUTING.md` from the `.github` repository. However, if a repository has any files in its own `.github/ISSUE_TEMPLATE` folder, such as issue templates or a `_config.yml` file, none of the contents of the default `.github/ISSUE_TEMPLATE` folder will be used. This allows repository maintainers to override the default files with specific templates or content on per-repository basis.
```

Default community health files are a set of predefined files that provide guidance and templates for maintaining a healthy and collaborative open source project.
You can add default community health files to a public repository called `.github`, in the root of the repository or in the `docs` or `.github` folders. These files help you automate and standardize various aspects of your project's development and community interaction. They are essential for promoting transparency, good practices, and collaboration in open source projects.

{% data variables.product.product_name %} will use and display default files for any repository owned by the account that does not have its own file of that type in the following order:

1. Local repository:
   1. The `.github` folder.
   1. The root of the repository.
   1. The `docs` folder.
1. The `.github` repository:
   1. The `.github` folder.
   1. The root of the repository.
   1. The `docs` folder.

> For example, anyone who creates an issue or pull request in a repository that does not have its own CONTRIBUTING file will see a link to the default CONTRIBUTING file. If a repository has any files in its own `.github/ISSUE_TEMPLATE` folder, including issue templates or a _config.yml_ file, none of the contents of the default `.github/ISSUE_TEMPLATE` folder will be used.

Author: Hope the newest changes are in line with what you were thinking.

> For example, anyone who creates an issue or pull request in a repository that does not have its own CONTRIBUTING file will see a link to the default CONTRIBUTING file. If a repository has any files in its own `.github/ISSUE_TEMPLATE` folder, including issue templates or a _config.yml_ file, none of the contents of the default `.github/ISSUE_TEMPLATE` folder will be used.

Author: I also do agree with the original description. I just want to make it even more clear how it works. Hope the changes are in line with your thoughts. If not, feel free to give specifics on what you want to see changed.
